src/sentry_clj/tracing.clj

```diff
@@ -0,0 +1,104 @@
+(ns sentry-clj.tracing
+  (:import
+    (io.sentry
+      CustomSamplingContext
+      EventProcessor
+      Hub
+      Scope
+      Sentry
+      SentryTraceHeader
+      SentryTracer
+      SpanStatus
+      TransactionContext)))
+
+(def span-status
+  {:ok SpanStatus/OK
+   :cancel SpanStatus/CANCELLED
+   :internal-error SpanStatus/INTERNAL_ERROR
+   :unknown SpanStatus/UNKNOWN
+   :unknown-error SpanStatus/UNKNOWN_ERROR
+   :invalid-argument SpanStatus/INVALID_ARGUMENT
+   :deadline-exceeded SpanStatus/DEADLINE_EXCEEDED
+   :not-found SpanStatus/NOT_FOUND
+   :already-exists SpanStatus/ALREADY_EXISTS
+   :permisson-denied SpanStatus/PERMISSION_DENIED
+   :resource-exhaused SpanStatus/RESOURCE_EXHAUSTED
+   :fail-precondition SpanStatus/FAILED_PRECONDITION
+   :aborted SpanStatus/ABORTED
+   :out-of-range SpanStatus/OUT_OF_RANGE
+   :unimplemented SpanStatus/UNIMPLEMENTED
+   :unavailable SpanStatus/UNAVAILABLE
+   :data-loss SpanStatus/DATA_LOSS
+   :unauthenticated SpanStatus/UNAUTHENTICATED})
+
+(def sentry-trace-header
```

Comment:
Would you consider making this private?

Reply:
I'm wondering if this definition is private or not, but I decide to treat as private.

I think if other web application library such as ring use the tracing feature, this definition could be defined in another file too.  However, it seems better to think if it happens, so I change to private def.